build/SandBoxAndPackageInParallel.proj

```diff
@@ -3,15 +3,15 @@
 
   <Import Project="$(MSBuildThisFileDirectory)/LinuxDistrosNativeInstaller.props" />
 
-  <Target Name="RunInSandBoxAndPackage" >
-      <MSBuild Projects="$(MSBuildThisFileDirectory)/SandBoxAndPackage.proj"
-      Targets="SandBoxAndPackage"
-            Properties="RepoRoot=$(RepoRoot);
-            LinuxDistrosNeedNativeInstaller=%(LinuxDistrosNeedNativeInstaller.Identity);
-            DockerFolder=%(LinuxDistrosNeedNativeInstaller.DockerFolder);
-            BuildConfiguration=$(BuildConfiguration);
-            InstallerOutputDirectory=$(InstallerOutputDirectory);
-            BuildInParallel='true'
-            " />
+  <Target Name="RunInSandBoxAndPackage">
+    <MSBuild Condition=" '%(LinuxDistrosNeedNativeInstaller.DockerFolder)' != '' "
```

Comment:
(minor) This check might not be needed anymore.

Reply:
Fixed...